docs/reference/rubrix_webapp_reference.rst

```diff
@@ -1,7 +1,7 @@
 .. _webapp_reference:
 
-Rubrix UI
-=========
+Rubrix webapp UI
+================
```

Comment:
I think it is more common to write "web app" (separated), a quick google search prefers this notation. Maybe we could drop Rubrix (it's kind of obvious) and just write:
```suggestion
Web App UI
==========
```